llama_stack/cli/model/remove.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import argparse
+import os
+import shutil
+import sys
+
+from llama_stack.cli.subcommand import Subcommand
+from llama_stack.distribution.utils.config_dirs import DEFAULT_CHECKPOINT_DIR
+
+
+def _ask_for_confirm(model) -> bool:
+    input_text = input(f"Are you sure you want to remove {model}? (y/n): ").strip().lower()
+    if input_text == "y":
```

Comment:
you don't need this function at all. you can simply change this to say:

```
if input("....").lower() == "y":
```
at the callsite